### PR TITLE
Added the prepare for develop script with volume mapping

### PR DESCRIPTION
**Pull request by mfurak**

### CODE REVIEW

File: prepare_for_develop.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+mkdir Zeeguu-Web/instance
+chmod uog+wrx Zeeguu-Web/instance
+
+docker stop zeeguu-web
+docker run --net=host -d --name=zeeguu-web-with-volume -v $(pwd)/Zeeguu-Web:/opt/Zeeguu-Web zeeguu-web
+
+echo "you can make changes to Zeeguu-Web locally; then execute 'docker exec zeeguu-web-with-volume python ./Zeeguu-Web/setup.py develop' to deploy"
```
Comment (mircealungu) [inline prepare_for_develop.sh]: @mfurak this is cool.
can we move the `docker exec zeeguu-web-with-volume python ./Zeeguu-Web/setup.py develop` into it's own shell script? something entitled like: './redeploy_web.sh' or so? then the echo message becomes shorter.

**Review by mircealungu — CHANGES_REQUESTED** [review]

**Comment by alinbalutoiu** [comment]
There are a couple changes to ` Zeeguu-Teacher-Dashboard-React` which I'm not sure if they are intended to be committed.

You could also change the script `rebuild_web.sh` to `run_web_develop.sh` where you run all those commands. Instead of the `echo` you can also do `docker run -it --net=host <all args here> python ./Zeeguu-Web/setup.py develop` which will start the docker container and run the command directly.

File: prepare_for_develop.sh (hunk 1 above)
Comment (mircealungu) [reply prepare_for_develop.sh]: Also, shouldn't this script also do with zeeguu-api-core what it does with zeeguu-web?

**Comment by mircealungu** [comment]
@mfurak : indeed, try to submit the PR without the changes to the Teacher-Dashboard. You seem to have retrieved the latest version of the Teacher-Dashboard-React locally in your lineup. However, you should normally keep the submodule versions that are published in the lineup when you submit PRs. (Ergo, don't recursivelly pull all submodules before sending a PR).

**Review by mircealungu — APPROVED** [review]
this finally works now

**Pull request merged**